client/src/main/scala/com.gu.contentapi.client/ContentApiClient.scala

```diff
@@ -162,14 +119,21 @@ trait ContentApiClientLogic {
       ThriftDeserializer.deserialize(response, StoriesResponse)
     }
 
-  /**
-   * Shutdown the client and clean up all associated resources.
-   *
-   * Note: behaviour is undefined if you try to use the client after calling this method.
-   */
-  def shutdown(): Unit = http.dispatcher().executorService().shutdown()
-
 }
 
-class GuardianContentClient(val apiKey: String) extends ContentApiClientLogic
-
+object ContentApiClient {
+  def item(id: String) = ItemQuery(id)
+  val search = SearchQuery()
+  val tags = TagsQuery()
+  val sections = SectionsQuery()
+  val editions = EditionsQuery()
+  val removedContent = RemovedContentQuery()
+  val atoms = AtomsQuery()
+  val recipes = RecipesQuery()
+  val reviews = ReviewsQuery()
+  val gameReviews = GameReviewsQuery()
+  val restaurantReviews = RestaurantReviewsQuery()
+  val filmReviews = FilmReviewsQuery()
+  val videoStats = VideoStatsQuery()
+  val stories = StoriesQuery()
+}
```

Comment:
👍 for moving these out of the class.
If backwards compatibility is important then we can make this a `trait` and let its companion object extend it, making it simpler for users to create a `new GuardianContentClient("") with ContentApiQueries`.